README.rst

```diff
@@ -206,10 +203,7 @@ If running in local mode, which uses Python's multiprocessing module, the
 ``local`` comms option and the number of workers must be specified. The script
 can then be run as a regular Python script::
 
-    python myscript.py
-
-These options may be specified via the command line by using the ``parse_args()``
-convenience function within libEnsemble's ``tools`` module.
+    python myscript.py --comms local --nworkers N
```

Comment:
Not sure we should remove this, as the added line assumes ``parse_args``. I do think that before it was a bit insufficient also, and maybe could do with a link to example of setting libE_specs for local comms. But as this is just an overview, maybe there is a concise way to express.

Reply:
Perhaps use the block from running libE in manual. https://libensemble.readthedocs.io/en/docs-jln_docs_updates/running_libE.html